v21.1/deploy-cockroachdb-with-kubernetes-openshift.md

```diff
@@ -0,0 +1,329 @@
+	~~~
+
+	~~~
+	#
+	# Welcome to the CockroachDB SQL shell.
+	# All statements must be terminated by a semicolon.
+	# To exit, type: \q.
+	#
+	# Server version: CockroachDB CCL v20.2.8 	(x86_64-unknown-linux-gnu, built 2021/04/23 13:54:57, go1.13.14) (same version as client)
+	# Cluster ID: 0813c343-c86b-4be8-9ad0-477cdb5db749
+	#
+	# Enter \? for a brief introduction.
+	#
+	root@crdb-tls-example-public:26257/defaultdb>
+	~~~
+
+	Now you can run SQL commands against the cluster.
+
+	{% include {{ page.version.version }}/orchestration/kubernetes-basic-sql.md %}
+
+If you can't access the SQL client, this may be related to your `--certs-dir` or `--host` flags. 
+
+1. Shell into the client pod and check for the necessary certs in the `--certs-dir` directory:
+
+	{{site.data.alerts.callout_success}}
+	You can also access the client pod by selecting it on the **Pods** page and clicking **Terminal**.
+	{{site.data.alerts.end}}
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc exec -it crdb-client-secure sh
+	~~~
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	# cd /cockroach/cockroach-certs
+	# ls
+	~~~
+
+	~~~
+	ca.crt           client.root.key
+	client.root.crt
+	~~~
+
+1. Check the name of the `public` service to use with the `--host` flag:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc get services
+	~~~
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)              AGE
+	crdb-tls-example          ClusterIP   None             <none>        26257/TCP,8080/TCP   14m
+	crdb-tls-example-public   ClusterIP   172.25.180.197   <none>        26257/TCP,8080/TCP   14m
+	~~~
+
+## Step 6. Access the DB Console
+
+To access the CockroachDB cluster's [DB Console](ui-overview.html):
+
+1. On secure clusters, [certain pages of the DB Console](ui-overview.html#db-console-access) can only be accessed by `admin` users.
+
+	Start the CockroachDB [built-in SQL client](cockroach-sql.html) from the client pod:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc exec -it crdb-client-secure -- ./cockroach sql --certs-dir=/cockroach/cockroach-certs/ --host=crdb-tls-example-public
+	~~~
+
+1.  Assign `roach` to the `admin` role (you only need to do this once):
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > GRANT admin TO roach;
+    ~~~
+
+1. Exit the SQL shell and pod:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > \q
+    ~~~
+
+1. In a new terminal window, port-forward from your local machine to the `crdb-tls-example-public` service:
+
+{% include copy-clipboard.html %}
+~~~ shell
+oc port-forward service/crdb-tls-example-public 8080
+~~~
+
+~~~
+Forwarding from [::1]:8080 -> 8080
+~~~
+
+1. Go to <a href="https://localhost:8080/" data-proofer-ignore>https://localhost:8080</a> and log in with the username and password you created earlier.
+
+    {% include {{ page.version.version }}/misc/chrome-localhost.md %}
```

Comment:
Could we add alternative instructions on how to download the ca.crt and make it trusted instead?